Gemfile

```diff
@@ -45,6 +45,7 @@ gem "stackprof"
 group :staging, :production do
   gem "newrelic_rpm"
   gem "rails_stdout_logging"
+  gem "scout_apm", "~> 2.0.x"
```

Comment:
I copy-past it from documentation, but now can't find from where. I'll remove it.